aspnetcore/mvc/razor-pages/index.md

```diff
@@ -373,10 +373,40 @@ To configure advanced options, use the extension method `AddRazorPagesOptions` o
 
 [!code-cs[main](index/sample/RazorPagesContacts/StartupAdvanced.cs?name=snippet_1)]
 
-Currently you can use the `RazorPagesOptions` to set the root directory for pages, or add application model conventions for pages. We hope to enable more extensibility this way in the future.
+Currently you can use the `RazorPagesOptions` to set the root directory for pages, or add application model conventions for pages. We'll enable more extensibility this way in the future.
 
 To precompile views, see [Razor view compilation](xref:mvc/views/view-compilation) .
 
 [Download or view sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/mvc/razor-pages/index/sample).
 
 See [Getting started with Razor Pages in ASP.NET Core](xref:tutorials/razor-pages/razor-pages-start), which builds on this introduction.
+
+### Specify that Razor Pages are at the content root
```

Comment:
Should we call out that by default Razor Pages are rooted to the `/Pages` directory.

Reply:
Yes ... good idea ... I'll do that.